### ISSUE TITLE
[Bug] Rule 0528 (too-few-calls-in-test-case) reports 0/1 keywords with templated test cases

### ISSUE THREAD

**Issue by antonpaa** [MarketSquare/robotframework-robocop#685]
### What happened?

The rule 0528 (`too-few-calls-in-test-case`) doesn't comply with templated test case structure. Instead, a standard error message is reported, just as below:
> [E] 0528 Test case 'Template Case' has too few keywords inside (0/1) (too-few-calls-in-test-case)

Sample code for reproducing:
``` robotframework
*** Settings ***
Documentation    Suite Documentation


*** Test Cases ***
Template Case
    [Documentation]    Documentation
    [Tags]    Tags
    [Template]    Do Stuff
    ${EMPTY}     ${EMPTY}
    log stuff    ${EMPTY}


*** Keywords ***
Do Stuff
    [Documentation]    Filler keyword
    [Arguments]    ${arg}    ${arg2}
    Log    ${arg}
```

### Robocop version

2.4.0

**Comment by ericbjones** [comment]
+1

**Comment by bhirsz** [comment]
Fixed in linked PR. Now template arguments will be counted as keyword calls (so we still detect the situation where there is no argument at all or too many template arguments). If someone wishes to ignore template tests in ``too-few-calls-in-test-case`` and ``too-many-calls-in-test-case`` it is possible to set ``ignore_templated`` for those rules to ``True``.

**Comment by bhirsz** [comment]
@antonpaaI Fixed and released in Robocop 2.5.0 yesterday - please check and provide the feedback if necessary :)
